static/templates/stream_settings/stream_types.hbs

```diff
@@ -80,3 +80,16 @@
     </div>
 </div>
 {{/if}}
+
+{{#if (or is_admin is_stream_edit)}}
+    <div id="push-notifications-enabled">
```

Comment:
I think we should disable this setting when the server is not configured to send push notifications.

Reply:
As mentioned in https://github.com/zulip/zulip/pull/23943#issuecomment-1409162400 I have not disabled it. If this should be disabled let me know I will make the changes.